app/controllers/application_controller.rb

```diff
@@ -94,18 +94,19 @@ def notary
   # Adapted from https://gist.github.com/redrick/2c23988368fb525c7e75
   # there is more there, including GeoIP which we may use as when we
   # address HT-1451
-  def log_action(log_entry, permitted_params)
+  def log_action(obj, permitted_params)
     raise UnfilteredParameters unless permitted_params.permitted?
 
+    raise "Unable to extract object id from #{obj.inspect}" if obj.id.blank?
     rails_action = "#{params[:controller]}##{params[:action]}"
-
+    log_entry = HTLog.new(model: obj.resource_type.to_s.camelize, objid: obj.id)
```

Comment:
I wonder if perhaps we should just pass `obj` to `HTLog.new` and have it be its responsibility to ask the object how it wants to be identified in the logs? e.g something like

```ruby
class HTLog

  def initialize(obj: nil, model: obj.resource_type.to_s.camelize, objid: obj.id)
    ...
  end

end
```

and

```
 Rails.logger.info "AUDIT LOG: #{log_entry.model}, #{log_entry.objid}, #{JSON.generate(log_entry.data)}"
```